package.json

```diff
@@ -47,6 +47,6 @@
     "mocha": "^10.0.0",
     "standard": "^17.0.0",
     "standard-markdown": "^7.1.0",
-    "typescript": "^5.0.3"
+    "typescript": "^5.4.5"
```

Comment:
since we are using the `^` (caret) operator, npm will automatically install the latest version of `typescript@5.* `, see https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004